ops/conda_env/sdist_test.yml

```diff
@@ -9,5 +9,3 @@ dependencies:
 - cmake
 - ninja
 - python-build
-- c-compiler
```

Comment:
Do we need to test installing with sdist?

Reply:
Yes, in the case, I messed up the packaging and broke the sdist.